server.js

```diff
@@ -32,6 +33,10 @@ app.get("/users/:id/responders", middleware.checkToken, user.getResponders);
 app.get("/users/:id/responders/count",middleware.checkToken,user.getResponderCount);
 app.delete("/users/:id/responders/:responderid", middleware.checkToken, user.deleteResponder);
 
+app.post("/metrics/alarm", alarmService.alarmStart)
```

Comment:
Do not use the service methods directly. Go through an Alarm Controller. The controller will be able to do all the validation checking on the request bodies and catch any errors thrown by your service methods. Let me know if this is not clear.